### ISSUE TITLE
ippusbxd crashing on boot up

### ISSUE THREAD

**Issue by tillkamppeter** [OpenPrinting/ippusbxd#32]
See [Ubuntu bug #1877095](https://bugs.launchpad.net/ubuntu/+source/ippusbxd/+bug/1877095):
```
ippusbxd 1.34-2 crashes on system boot in Focal Fossa Ubuntu 20.04. Kernel 5.4.0-29.
ippusbxd also crashes when booting from an install media USB Stick (Ubuntu 20.04).
This occurs on Desktop PC and laptops connected to an HP Envy 6200 series Printer via USB cable.
Disconnecting the USB cable to the printer, the crash does not occur on boot up.
Earlier releases of Ubuntu (eg 19.10 and 19.04) boot cleanly.

report :
systemctl --failed
  UNIT LOAD ACTIVE SUB DESCRIPTION
ippusbxd@001:003.service loaded failed failed Daemon to make IPP-over-USB printers available as network printers (001:003)

---
ProblemType: Bug
ApportVersion: 2.20.11-0ubuntu27
Architecture: amd64
CasperMD5CheckResult: skip
CurrentDesktop: MATE
DistroRelease: Ubuntu 20.04
InstallationDate: Installed on 2020-05-04 (2 days ago)
InstallationMedia: Ubuntu-MATE 20.04 LTS "Focal Fossa" - Release amd64 (20200423)
NonfreeKernelModules: nvidia_modeset nvidia
Package: ippusbxd 1.34-2ubuntu1
PackageArchitecture: amd64
ProcVersionSignature: Ubuntu 5.4.0-29.33-generic 5.4.30
Tags: focal
Uname: Linux 5.4.0-29-generic x86_64
UpgradeStatus: No upgrade log present (probably fresh install)
UserGroups: adm cdrom dip lpadmin lxd plugdev sambashare sudo vboxusers
_MarkForUpload: True
```
In the comments [#10](https://bugs.launchpad.net/ubuntu/+source/ippusbxd/+bug/1877095/comments/10) and [#11](https://bugs.launchpad.net/ubuntu/+source/ippusbxd/+bug/1877095/comments/11) one can see the backtrace of the crash. It seems to be in the poll of eSCL scanner capabilities.
@Ordissimo, could you have a look? Thanks.

**Comment by ThierryHFR** [comment]
I'm watching.

**Comment by ThierryHFR** [comment]
This should solve this problem

**Comment by tillkamppeter** [comment]
Fixed by pull request #33 
Thank you very much.
